molecule/resources/converge.yml

```diff
@@ -21,5 +21,6 @@
       changed_when: false
       failed_when: systemctl_status.rc > 1
 
-  roles:
-    - mirsg.role_name
+  # TODO: change this to the desired role name
+  # roles:
+  #   - mirsg.role_name
```

Comment:
`ansible-lint` will give a `syntax-check[specific]` error here. `noqa: syntax-check[specific]` will not work here without modifying the `.ansible-lint` config/`pre-commit` flags to make that rule `skippable` https://ansible.readthedocs.io/projects/lint/usage/#output-formats. Commenting out seems like a better approach.

Reply:
what is the error it gives? Is it the same as this https://github.com/UCL-MIRSG/ansible-role-template/issues/10

Reply:
```
syntax-check[specific]: the role 'mirsg.role_name' was not found in /Users/paddy/arc/mirsg/ansible-role-template/molecule/resources/roles:/Users/paddy/.cache/ansible-compat/933984/roles:/Users/paddy/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/Users/paddy/arc/mirsg/ansible-role-template/molecule/resources
```

Reply:
Essentially you can't install something that doesn't exist with `ansible-galaxy`

Reply:
Ah, didn't think that would work. Seeing as this is a template, I'll go with the first option for now.